__tests__/app.test.js

```diff
@@ -1,47 +1,87 @@
 const seed = require(`${__dirname}/../db/seeds/seed.js`);
 const request = require("supertest");
 
 beforeEach(() => {
-  return seed(Data);
+  return seed(data);
 });
 
 afterAll(() => connection.end());
 
-describe("/api/categories", () => {
-  it("200: GET /api/categories should respond with array of category objects", () => {
-    return request(app)
-      .get("/api/categories")
-      .expect(200)
-      .then(({ body }) => {
-        const { categories } = body;
-
-        expect(categories).toBeInstanceOf(Array);
-        categories.forEach((category) => {
-          expect(category).toBeInstanceOf(Object);
+describe("nc_games_test", () => {
+  describe("/api/categories", () => {
+    it("200: GET /api/categories should respond with array of category objects", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          const { categories } = body;
+          expect(categories).toBeInstanceOf(Array);
+          categories.forEach((category) => {
+            expect(category).toBeInstanceOf(Object);
+          });
         });
-      });
-  });
-  it("200: GET /api/categories should have expected shape", () => {
-    return request(app)
-      .get("/api/categories")
-      .then(({ body }) => {
-        const { categories } = body;
-        expect(categories.length).toBe(4);
-        categories.forEach((category) => {
-          expect(category).toHaveProperty("slug", expect.any(String));
-          expect(category).toHaveProperty("description", expect.any(String));
+    });
+    it("200: GET /api/categories should have expected shape", () => {
+      return request(app)
+        .get("/api/categories")
+        .then(({ body }) => {
+          const { categories } = body;
+          expect(categories.length).toBe(4);
+          categories.forEach((category) => {
+            expect(category).toHaveProperty("slug", expect.any(String));
+            expect(category).toHaveProperty("description", expect.any(String));
+          });
+        });
+    });
+    it("404: Should respond with invalid path message if given invalid path input", () => {
+      return request(app)
+        .get("/api/bananas")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("404 path not found");
         });
-      });
+    });
   });
-  it("404: Should respond with invalid path message if given invalid path input", () => {
-    return request(app)
-      .get("/api/bananas")
-      .expect(404)
-      .then(({ body }) => {
-        expect(body.msg).toBe("404 path not found");
-      });
+  describe("/api/reviews/:review_id", () => {
+    it("200: GET /api/reviews/:review_id should return relevant review_id data", () => {
+      return request(app)
+        .get("/api/reviews/3")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews).toBeInstanceOf(Object);
+          expect(reviews).toEqual({
```

Comment:
.toMatchObject or .objectContaining is better than .toEqual when checking what you expect an object to look like, even when it's a specific object